source/scss/_pf-variables.scss

```diff
@@ -580,86 +713,54 @@ $card-img-overlay-padding: 1.25rem !default;
 $card-deck-margin:         .625rem !default;
 $card-columns-sm-up-column-gap: 1.25rem !default;
 
```

Comment:
yes they should, but since cards will not be in the first alpha release I'll revisit card variables when we include them